src/UI/components/about/aboutTeamPanel.tsx

```diff
@@ -15,34 +16,95 @@ export default function AboutTeamPanel({
   imageURL: string;
   description: string;
 }) {
-  return (
+  const theme = useTheme();
+  const isMatch = useMediaQuery(theme.breakpoints.down('sm'));
+  return isMatch ? (
```

Comment:
Looks like the only difference in the two code blocks is some styling - instead of having the components written out twice, would it be possible to just use `isMatch` to set the style of the individual components?